cli/download_source.go

```diff
@@ -95,6 +95,7 @@ func downloadTerraformSourceIfNecessary(terraformSource *TerraformSource, terrag
 	// before and after hooks (if any).
 	terragruntOptionsForDownload := terragruntOptions.Clone(terragruntOptions.TerragruntConfigPath)
 	terragruntOptionsForDownload.TerraformCommand = CMD_INIT_FROM_MODULE
+	terragruntOptionsForDownload.Logger = terragruntOptions.Logger
```

Comment:
Does `Clone` not do this automatically?

Reply:
the clone function sets up a new logger logging to the current ErrWriter.. which is not where the current logger is logging to.. 

I know it is not the cleanest solution to just set it this way, but since it fixed it for me, maybe an actual go developer can come up with a cleaner implementation.

Reply:
btw it seems that ErrWriter and Logger are not really connected in a logical consequent way. since setting ErrWriter does not change the Logger in any way and also not sets up a new Logger.

Reply:
Hm, it seems like the right fix would be to make the `Clone` method do the right thing. This small workaround means (a) that `Clone` may still be doing the wrong thing in other places and (b) if we make updates to `Clone` in the future, that may break in this one-off fix here.